psa-crypto-sys/src/utils.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+#![allow(non_snake_case)]
+/// Additional functionality required that Mbed Crypto does not provide
```

Comment:
Maybe it is "PSA Crypto" does not provide?
Also it might be good to rename this file `extra.rs`. `utils.rs` often ends up being a pot-pourri of non-sorted things 😃